project-base/tests/App/Functional/Model/Payment/IndependentPaymentVisibilityCalculationTest.php

```diff
@@ -116,8 +116,17 @@ public function getDefaultPayment($enabledForDomains, $hidden)
         }
         $paymentData->name = $names;
         $paymentData->hidden = $hidden;
-        $paymentData->enabled = $enabledForDomains;
+        $paymentData->enabled = $this->getFilteredEnabledForDomains($enabledForDomains);
 
         return new Payment($paymentData);
     }
+
+    /**
+     * @param array $enabledForDomains
+     * @return array
+     */
+    private function getFilteredEnabledForDomains(array $enabledForDomains): array
```

Comment:
This method is very hard for reading to me. It might help to assign the second parameter to a new variable, what do you think?

Reply:
I think it is ok, as it strips field for domains which are not configured. Maybe the method name could be better.